index.html

```diff
@@ -319,5 +320,21 @@ <h3><i class="lni lni-envelope"></i><strong>Leave</strong> a Message.</h2>
       </section>
     </main>
     </main>
+
+    <footer>
+
+      <img src="assets/img/logo-2.svg">
+      <div class="social_media_icons">
+        <a href="#"><i class="fab fa-facebook-f" href="#"></i></a> 
+        <a href="#"><i class="fab fa-twitter"></i></a>
+        <a href="#"><i class="fab fa-instagram"></i></a>
+        <a href="#"><i class="fab fa-linkedin"></i></a>
+      </div>
+      <a href="tel:+8801234567890">+8801234567890</a>
+      <p><a href="mailto:support@uideck.com">support@uideck.com</a></p>
+      <p>Designed by <a href="">UIdeck</a> and Built-with <a href=""> Ayro UI</a></p>
```

Comment:
* [] no empty href, add link
* [] remove space before ` Ayro UI` between anchor tags and place outside of tags, e.g. right after 'Built-with'